flows/atd-knack-banner/knack_banner.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+"""
+Name: Knack Banner HR App integration
+Description: Update knack HR app based on records in Banner and CTM
+Schedule: "45 13 * * *"
+
+prefect deployment build flows/atd-knack-banner/knack_banner.py:knack_hr_banner_flow -t production \
+    --cron "45 13 * * *" -q atd-data-03 --name "HR Knack Banner" -o "deployments/knack_banner.yaml" \
+    -sb github/atd-prefect-main-branch --skip-upload
+
+"""
+
+import prefect
+import docker
+
+# Prefect
+from prefect import flow, task, get_run_logger
+from prefect.blocks.system import JSON
+
+docker_image = f"atddocker/atd-knack-banner:production"
+
+
+@task(name="get_environment_variables")
+def get_env_vars():
+    # Environment Variables stored in JSON block in Prefect
+    return JSON.load("atd-knack-banner")
+
+
+# Retrieve the provider's data
+@task(
+    name="update HR employees in knack",
+    retries=2,
+    retry_delay_seconds=300,
+    task_run_name="atd-knack-banner/update-employees",
```

Comment:
Does this do anything besides giving it a name? I had a hard time wrapping my head around this parameter when I was reading through the docs.

Reply:
i dont think it does, I thought it would show up in the UI, but i think i named everything too similarly so I can't tell